app/views/provider_interface/hesa_export/new.html.erb

```diff
@@ -17,7 +17,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">
-      <%= t('page_titles.provider.export_hesa_data') %>
+      <%= t('page_titles.provider.export_hesa_data', timeframe: "since #{EndOfCycleTimetable::CYCLE_DATES[RecruitmentCycle.current_year][:apply_reopens].to_s(:govuk_date)}") %>
```

Comment:
Maybe consider adding a method to `EndOfCycleTimetable` to return a `Date` instead of putting this quite gnarly  hash business out here?